pkg/quarkus/v1alpha/scaffolds/internal/templates/makefile.go

```diff
@@ -91,6 +101,17 @@ docker-build: ## Build docker image with the manager.
 docker-push: ## Push docker image with the manager.
 	mvn package -Dquarkus.container-image.push=true -Dquarkus.container-image.image=${IMG}
 
+##@Bundle
+bundle-generate:  
```

Comment:
This should be 

```suggestion
.PHONY: bundle
bundle:  ## Generate bundle manifests and metadata, then validate generated files.
```

We want consistent makefile targets with the other projects. So it should be `build`. We also need `.PHONY` lines for the other targets in the Makefile (new and existing).

Reply:
Yes, agree. Updated according to above comments